src/mmw/js/src/modeling/tr55/runoff/views.js

```diff
@@ -45,9 +46,9 @@ var ResultView = Marionette.ItemView.extend({
                     depDisplayNames: ['Infiltration', 'Runoff', 'Evaporation']
                 };
 
-            if (this.scenario.get('is_current_conditions')) {
+            if (this.scenario.get('is_current_conditions') || this.compareMode) {
                 data = [
-                    getBarData('', 'unmodified'),
+                    getBarData('', 'modified'),
```

Comment:
Is the reasoning here that both `unmodified` and `modified` are identical for Current Conditions so it doesn't matter for them, and by choosing `modified` as the single bar-chart we ensure that when in `compareMode` the chart reflects the modifications?